<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="7" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Example test run" duration-ms="265629" started-at="2020-05-31T17:36:55 CEST" finished-at="2020-05-31T17:41:20 CEST">
    <groups>
    </groups>
    <test name="Simple Test" duration-ms="265629" started-at="2020-05-31T17:36:55 CEST" finished-at="2020-05-31T17:41:20 CEST">
      <class name="com.example.Main">
        <test-method status="PASS" signature="test1_StartWebDriver()[pri:0, instance:com.example.Main@5bb21b69]" name="test1_StartWebDriver" duration-ms="5980" started-at="2020-05-31T17:36:55 CEST" finished-at="2020-05-31T17:37:01 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1_StartWebDriver -->
        <test-method status="PASS" signature="test2_Login()[pri:0, instance:com.example.Main@5bb21b69]" name="test2_Login" duration-ms="40291" started-at="2020-05-31T17:37:01 CEST" finished-at="2020-05-31T17:37:41 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2_Login -->
        <test-method status="FAIL" signature="test3_AddNewProject()[pri:0, instance:com.example.Main@5bb21b69]" name="test3_AddNewProject" duration-ms="87188" started-at="2020-05-31T17:37:41 CEST" finished-at="2020-05-31T17:39:08 CEST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [3]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [3]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at com.example.Main.test3_AddNewProject(Main.java:96)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3_AddNewProject -->
        <test-method status="PASS" signature="test4_SelectProject()[pri:0, instance:com.example.Main@5bb21b69]" name="test4_SelectProject" duration-ms="28832" started-at="2020-05-31T17:39:08 CEST" finished-at="2020-05-31T17:39:37 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test4_SelectProject -->
        <test-method status="PASS" signature="test5_AddDataToSelectedProject()[pri:0, instance:com.example.Main@5bb21b69]" name="test5_AddDataToSelectedProject" duration-ms="48461" started-at="2020-05-31T17:39:37 CEST" finished-at="2020-05-31T17:40:25 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test5_AddDataToSelectedProject -->
        <test-method status="PASS" signature="test6_EditDataFromSelectedProject()[pri:0, instance:com.example.Main@5bb21b69]" name="test6_EditDataFromSelectedProject" duration-ms="25485" started-at="2020-05-31T17:40:25 CEST" finished-at="2020-05-31T17:40:51 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test6_EditDataFromSelectedProject -->
        <test-method status="PASS" signature="test7_DeleteDataFromSelectedProject()[pri:0, instance:com.example.Main@5bb21b69]" name="test7_DeleteDataFromSelectedProject" duration-ms="29358" started-at="2020-05-31T17:40:51 CEST" finished-at="2020-05-31T17:41:20 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test7_DeleteDataFromSelectedProject -->
      </class> <!-- com.example.Main -->
    </test> <!-- Simple Test -->
  </suite> <!-- Example test run -->
</testng-results>
